.circleci/config.yml

```diff
@@ -223,15 +224,15 @@ jobs:
     executor: main-executor
     working_directory: ~/react-widgets
     environment:
-      - AWS_BUCKET: "code.s4d.io"
-      - AWS_REGION: "us-east-1"
+      AWS_BUCKET: code.s4d.io
```

Comment:
I already put the bucket in the env on the dashboard
The region can also go there

Reply:
@taymoork2 the bucket and region were not in the web config until this morning when I added it. I prefer them to be in the config file instead since they are not sensitive data.